C/CoinUtils/build_tarballs.jl

```diff
@@ -0,0 +1,82 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "CoinUtils"
+version = v"2.11.4"
+
+# Collection of sources required to complete build
+sources = [
+    # GitSource("https://github.com/coin-or/CoinUtils.git", "d4f2b7f1897b67da6929ab42aa6b1962a388c5b9"),
+    ArchiveSource("https://github.com/coin-or/CoinUtils/archive/releases/2.11.4.tar.gz",
+                  "d4effff4452e73356eed9f889efd9c44fe9cd68bd37b608a5ebb2c58bd45ef81"),
+    # DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/CoinUtils-releases-2.11.4
+
+# Remove wrong libtool files
+rm -f /opt/${target}/${target}/lib*/*.la
+
+# if [[ "${target}" == *-musl* ]]; then
+#     # This is to fix the following error:
+#     #    node_heap.cpp:11:22: fatal error: execinfo.h: No such file or directory
+#     #     #include <execinfo.h>
+#     # `execinfo.h` is GlibC-specific, not Linux-specific
+#     atomic_patch -p1 "${WORKSPACE}/srcdir/patches/glibc_specific.patch"
+# fi
```

Comment:
Perhaps we can get the musl platforms by enabling this, and copying the patch from the other CoinUtils PR that @giordano had prepared.

Reply:
The patch doesn't apply unfortunately.

Reply:
Where is the file supposed to be patched?